app/lib/server/routers/docs.router.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const routesAbstractionController = require('../controllers/RoutesAbstraction.controller.js').getInstance();
+
+module.exports = {
+    routesAbstractionController,
+    path: '/docs',
```

Comment:
You should name this API docs, because maybe at some point you will want to add more generic doc, for example on how to use RCT

Reply:
all endpoints in WebUi like this are prefixed with /api, so it will be /api/docs

Reply:
aplied